src/west/commands/build.py

```diff
@@ -188,6 +189,20 @@ def _setup_source_dir(self):
             source_dir = os.getcwd()
         self.source_dir = os.path.abspath(source_dir)
 
+    def _get_available_boards_str(self):
+        path = Path(os.getenv("ZEPHYR_BASE")+"/boards")
```

Comment:
I've never used pathlib. It looks really nice and convenient. Maybe we'll switch to it someday in west! But for now, we just use os.path -- can you please keep that consistent and use that too?